app/controllers/action_officers_controller.rb

```diff
@@ -45,6 +45,7 @@ def update
     end
   end
 
+  # TODO I don't think this is used
```

Comment:
This is currently used in order to power the autocomplete menu (see `find_action_officers` route)